asyncpg/connect_utils.py

```diff
@@ -258,7 +259,7 @@ def _dot_postgresql_path(filename) -> pathlib.Path:
 
 def _parse_connect_dsn_and_args(*, dsn, host, port, user,
                                 password, passfile, database, ssl,
-                                connect_timeout, server_settings):
+                                tls_proxy, connect_timeout, server_settings):
```

Comment:
On a second thought, I'm not liking the naming of the argument.  This really isn't about a proxy, rather whether we use `start_tls` or establish a TLS connection straight away.  Perhaps `direct_tls` or `straight_tls`?

Reply:
Agreed, updated param to `direct_tls`